### DISCUSSION TITLE
Use AsyncFriendlyStackTrace github package

### DISCUSSION THREAD

**Discussion by ArnaudB88** [RehanSaeed/Serilog.Exceptions#28] [Q&A]
Hi,
Stacktraces of async functions are not so readable. Is it possible to use the specified github package and change the code in ExceptionDestructurer.Destructure() so stacktraces are more readable.
Thanks!

**Comment by RehanSaeed** [comment]
While I think that project is cool, it is bad performance wise. If an app throws lots of exceptions, we don't want to affect app performance just to log an error in a nicer format.
